cmd/test/k8sValidVersions.go

```diff
@@ -0,0 +1,111 @@
+package test
```

Comment:
please format both files that changed to pass the linter.
In your IDE you should have an option to format the code, if you need help with that - let me know

Reply:
Actually I was using Vim editor for this. Don't know, if it has any options for formatting.
I will format it in the vscode now :)